### ISSUE TITLE
Scan anything screen

### ISSUE THREAD

**Issue by 1010Tom** [get10101/10101#74]
Depends on #59

**Comment by bonomat** [comment]
Still on our list once we focus more on the wallet.

**Comment by klochowicz** [comment]
it's a fairly low-hanging fruit - we had a QR scanner in 10101-PoC (this one would only different in that it would try to parse both kinds of invoices and dispatch appropriate action.

**Comment by Restioson** [comment]
When you say 'both kinds' of invoices - which kinds are there?

**Comment by holzeis** [comment]
There are actually 3 kinds of "invoices".

- on chain (an address)
- lightning
- synthetic usd (lightning with a hint that the received lightning should get immediately stabilised)

**Comment by klochowicz** [comment]
I would keep the 3rd kind of invoices (synthetic USD) out of scope of the initial screen, with ability to bring them in later on, because that would need more work.
